src/uniprotkb/components/statistics/styles/historical-release-entries-line-plot.module.scss

```diff
@@ -0,0 +1,9 @@
+@import 'franklin-sites/src/styles/colours';
+
+.line-plot {
+  path.line {
+    fill: none;
+    stroke: $colour-weldon-blue;
```

Comment:
Would be good to use the global custom properties for colours here, instead of having to import from franklin.